docs/documentation.md

```diff
@@ -0,0 +1,159 @@
+# Documentation
+
+Available commands: 
+
+- [`workon`](#workon)
+- [`wrapup`](#wrapup)
+- [`deliver`](#deliver)
+- [`auth`](#auth)
+- [`issues`](#issues)
+- [`web`](#web)
+- [`mkalias`](#mkalias)
+- [`cleanup`](#cleanup)
+- [`config` ](#config)
+- [global flags](#global-flags)
+
+## requirements
+- Activate issues on the designated GitHub repository
+- Create the labels used by the phlow. Find the labels [here](https://www.praqma.com/stories/a-pragmatic-workflow/)
+- Download and install git-phlow
+- Start creating tasks in GitHub issues
+
+
+### workon
+```
+git phlow workon [issue]
+```
+**flags**
+
+`-t`, `--target` - INI block in .gitconfig
+
+**desc**
+
+Creates a new local branch from a remote integration branch, based on an issue from the chosen issue management system [See configuration]. 
+
+### wrapup
+```
+git phlow wrapup
+```
+**flags**
+
+`-t`, `--target` - INI block in .gitconfig
+
+`--force` - use a custom commit message instead of the auto-generated
+
+**desc**
+
+Add changes from the workspace to the index/staging area and commits with `smart commit`syntax that will close the mentioned issue when it is integrated on the integration branch.
```

Comment:
spacing after smart commit. Also example with description of state before and after command